functions/fish_prompt.fish

```diff
@@ -1172,6 +1189,7 @@ function fish_prompt -d 'bobthefish, a fish theme optimized for awesome'
     __bobthefish_prompt_nix
     __bobthefish_prompt_desk
     __bobthefish_prompt_rubies
+    __bobthefish_prompt_golang
     __bobthefish_prompt_virtualfish
     __bobthefish_prompt_virtualgo
```

Comment:
should this be combined with virtualgo? e.g. the `rubies` segment uses whatever is most relevant of the many ways to define a ruby version.

Reply:
I'd be down with that; I'll probably also see if I can steal the "only show Ruby version when in a folder where it matters" bit from the `rubies` segment too ( more thoughts on this in [my other comment](https://github.com/oh-my-fish/theme-bobthefish/pull/327#issuecomment-1155525636) )